restdocs-api-spec-jsonschema/src/test/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGeneratorTest.kt

```diff
@@ -138,6 +141,15 @@ class JsonSchemaFromFieldDescriptorsGeneratorTest {
         whenSchemaGenerated()
 
         then(schema).isInstanceOf(ArraySchema::class.java)
+        then((schema as ArraySchema).allItemSchema).isInstanceOf(CombinedSchema::class.java)
+        val combinedSchema = ((schema as ArraySchema).allItemSchema) as CombinedSchema
+        then(combinedSchema.criterion).isEqualTo(ONE_CRITERION)
```

Comment:
`ONE_CRITERION` seems to be the `oneOf` criterion:
https://github.com/everit-org/json-schema/blob/master/core/src/main/java/org/everit/json/schema/CombinedSchema.java#L119

This also matches current behavior when generating JSON schema for arrays with unknown items type:
https://github.com/ePages-de/restdocs-api-spec/issues/147